examples/step-3/doc/results.dox

```diff
@@ -138,6 +138,63 @@ suggestions:
   discontinuous boundary values, zero on three sides of the square, and one on
   the fourth.
 
+  <li>
+  Use triangles: As mentioned in the results section of step-1, for
+  historical reasons, almost all tutorial programs for deal.II are
+  written using quadrilateral or hexahedral meshes. But deal.II also
+  supports triangular and tetrahedral meshes. So a good experiment would
+  be to replace the mesh used here by a triangular mesh.
+
+  This is *almost* trivial. First, as discussed in step-1, we may want
+  to start with the quadrilateral mesh we are already creating, and
+  then convert it into a triangular one. You can do that by replacing
+  the first line of `Step3::make_grid()` by the following code:
+  @code
+  Triangulation<2> triangulation_quad;
+  GridGenerator::hyper_cube(triangulation_quad);
```

Comment:
I think you want to choose
```suggestion
  GridGenerator::hyper_cube(triangulation_quad, -1, 1);
```
This should then also fix the difference in the solution magnitude you observe.

Reply:
Oh, doh!

Reply:
That indeed fixes the problem, just for completeness.